ee/desktop/runner/runner.go

```diff
@@ -113,23 +130,24 @@ type DesktopUsersProcessesRunner struct {
 	runnerServer *runnerserver.RunnerServer
 }
 
-// processRecord is used to track spawned desktop processes.
+// ProcessRecord is used to track spawned desktop processes.
 // The path is used to ensure another process has not taken the same pid.
 // The existence of a process record does not mean the process is running.
 // If, for example, a user logs out, the process record will remain until the
 // user logs in again and it is replaced.
-type processRecord struct {
-	process    *os.Process
-	path       string
-	socketPath string
+type ProcessRecord struct {
```

Comment:
I don't think you need to export this.

Reply:
I always forget about that, my brain can't handled an unexported struct and exported properties